recipes/spix/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+        return {
+            "Visual Studio": "14",
+            "gcc": "5",
+            "clang": "3.4",
+            "apple-clang": "10"
+        }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("anyrpc/1.0.2")
+        if self.options.qt_major == 5:
+            self.requires("qt/5.15.6")
+        else:
+            self.requires("qt/6.3.1")
+        self.requires("expat/2.4.9", override=True)
+        
+    def validate(self):
+        if self.info.settings.compiler.cppstd:
+            check_min_cppstd(self, self._minimum_cpp_standard)
+        minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
+        if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._minimum_cpp_standard}, which your compiler does not support."
+            )
+
+        if self.options.qt_major == 6 and not self.options["qt"].qtshadertools:
+            raise ConanInvalidConfiguration(f"{self.ref} requires qt:qtshadertools to get the Quick module")
+        if not (self.options["qt"].gui and self.options["qt"].qtdeclarative):
+            raise ConanInvalidConfiguration(f"{self.ref} requires qt:gui and qt:qtdeclarative to get the Quick module")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["SPIX_BUILD_EXAMPLES"] = False
+        tc.variables["SPIX_BUILD_TESTS"] = False
+        tc.variables["SPIX_QT_MAJOR"] = self.options.qt_major
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+        if self.options.qt_major == 6:
+            replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"), "set(CMAKE_CXX_STANDARD 14)", "set(CMAKE_CXX_STANDARD 17)")
+
+    def build(self):
+        self._patch_sources()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="LICENSE.txt", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
```

Comment:
This is a good sign you are missing some `package_info`. Check the project you indeed are https://github.com/faaxm/spix/blob/4de44072d76b49ceaae9c57046f2260948c065a4/lib/CMakeLists.txt#L147